LoopKit/Service/VersionCheckService.swift

```diff
@@ -12,21 +12,29 @@ import Foundation
 public enum VersionUpdate: Comparable, CaseIterable {
     /// No version update needed.
     case noneNeeded
+    /// A update is available, but it is just informational (supported update).
+    case updateAvailable
```

Comment:
[nit] The name seems inconsistent to me. None of the other cases use the `update` noun because it is implied in the type name. Also adding in the change in terminology from "supported" to "recommended". Consider either:
```
    case noUpdateAvailable:
    case updateAvailable:
    case recommendedUpdateNeeded:
    case requiredUpdateNeeded:
```
or 
```
    case none:
    case available:
    case recommended:
    case required:
```
Frankly, I prefer the second list.